.github/workflows/TEST-release.yml

```diff
@@ -42,7 +42,7 @@ jobs:
       - run: sudo chmod -R 777 $ARTIFACTS_DIR
       - run: mkdir -p $CUCUMBER_ARTIFACTS_DIR
       - run: sudo chmod -R 777 $CUCUMBER_ARTIFACTS_DIR
-      - run: npm run test:bdd:release;
+      - run: npm run test:bdd;
```

Comment:
Please update the name of github action to TEST-bdd. Both in file and file naming.